roles/arteria-siswrap-ws/defaults/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+arteria_install_path: "{{ root_path }}/sw/arteria"
+arteria_siswrap_env_root: "{{ arteria_install_path }}/siswrap_venv"
+arteria_siswrap_config_root: "{{ ngi_pipeline_conf }}/arteria/siswrap"
+arteria_siswrap_app_config: "{{ arteria_siswrap_config_root }}/app.config"
+arteria_siswrap_logger_config: "{{ arteria_siswrap_config_root }}/logger.config"
+
+arteria_siswrap_port: 10430
+arteria_siswrap_mail_from: johan.hermansson@medsci.uu.se
```

Comment:
Kind of odd that the siswrap is mailing from your personal account rather than a more dedicated entry.